src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -273,6 +288,12 @@ export const validateAddress = (
         confirmedSuggestions,
       });
     }
+    const suggestedAddressSelected = getSelectedAddressId(getState());
+    recordEvent({
+      event: 'profile-transaction',
+      'profile-section': analyticsSectionName,
+      'profile-addressSuggestionUsed': suggestedAddressSelected ? 'yes' : 'no',
```

Comment:
won't we just want to hard code this as `'no'` in this case since the user didn't manually pick a suggested address since they never even saw the address validation modal?